python/cog/server/worker.py

```diff
@@ -184,10 +177,12 @@ def _setup(self) -> None:
             if hasattr(self._predictor, "setup"):
                 run_setup(self._predictor)
         except Exception as e:
+            capture_exception(e) # Cpaturing exception with sentry
```

Comment:
Were these not getting caught by Sentry? How did we verify this?

Reply:
Sentry only captures unhandled exceptions, that's why we need to manually report them.
I tested locally by directly raise exception on both predictor's `setup` and `predict` methods.